dev/com.ibm.ws.microprofile.rest.client_fat/publish/servers/mpRestClient11.cdiPropsAndProviders/bootstrap.properties

```diff
@@ -0,0 +1,10 @@
+com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.jaxrs*=all:org.apache.cxf.*=all
```

Comment:
we should limit logging to `*=info` unless you already suspect that there is a problem